packages/client/Atmosphere.ts

```diff
@@ -104,7 +104,8 @@ export default class Atmosphere extends Environment {
   upgradeTransportPromise: Promise<void> | null = null
   // it's only null before login, so it's just a little white lie
   viewerId: string = null!
-  userId: string | null = null // DEPRECATED
+  /** @deprecated */
```

Comment:
By using the `jsdoc`-style syntax, many IDEs will mark uses of this property with a strikethrough:

![Screen Shot 2022-02-01 at 2 26 39 PM](https://user-images.githubusercontent.com/630449/152054106-2bd5ae8c-4f9c-4ebf-ba34-ce19876e33f3.png)

This is a nice visual indicator that makes me think twice before using it.